pkg/minikube/reason/reason.go

```diff
@@ -192,6 +192,7 @@ var (
 	HostHomeChown        = Kind{ID: "HOST_HOME_CHOWN", ExitCode: ExHostPermission}
 	HostBrowser          = Kind{ID: "HOST_BROWSER", ExitCode: ExHostError}
 	HostConfigLoad       = Kind{ID: "HOST_CONFIG_LOAD", ExitCode: ExHostConfig}
+	HostPermissionDenied = Kind{ID: "HOST_PERMISSION_DENIED", ExitCode: ExHostPermission}
```

Comment:
This looks basically good, but can get better if we remove the duplicate data within `known_issues.go`.  

Please rename this to `HostHomePermission` and copy the "HOST_HOME_PERMISSION" `Kind{}` object from: 

https://github.com/kubernetes/minikube/blob/3f9c6d10d9cd4a56abe436f6963d5edb5a00c67e/pkg/minikube/reason/known_issues.go#L205

This will then provide the user appropriate advice if they run into this error. Your exit code, ExHostPermission is the correct one to use, however.

Then, replace the `HOST_HOME_PERMISSION Kind{}` within `known_issues.go` with a reference to `HostHomePermission`. This way we may also provide the correct feedback for any other errors which match this regexp in the future.